src/vs/workbench/contrib/terminal/browser/terminalProcessManager.ts

```diff
@@ -230,6 +234,27 @@ export class TerminalProcessManager extends Disposable implements ITerminalProce
 		return undefined;
 	}
 
+	// Fetch any extension environment additions and apply them
+	private async _refreshEnvironmentVariables(activeWorkspaceRootUri: URI | undefined, shellLaunchConfig: IShellLaunchConfig): Promise<void> {
```

Comment:
Since this is only called when a terminal is created let's name it `_setupEnv...` instead of refresh.